src/app/services/data.store.ts

```diff
@@ -4,22 +4,18 @@ import { BehaviorSubject } from 'rxjs';
 @Injectable({
   providedIn: 'root'
 })
-export class DataStoreService {
+export class DataStore {
 
-  private _users = new BehaviorSubject(undefined);
+  private _users = new BehaviorSubject('');
```

Comment:
Do not use empty string here since the result is always going to be an array of users. So, use `[]`